src/Service/LoggingServiceFactory.php

```diff
@@ -32,9 +32,9 @@ public function __construct(
         MapperFactory $mapperFactory,
         LoggerInterface $logger,
         Stopwatch $stopwatch,
-        CacheItemPoolInterface $cacheItemPoolInterface
+        CacheItemPoolInterface $cacheItemPool
```

Comment:
A little thing I missed last time -  The LoggingServiceFactory mimics the constructor of the ServiceFactory, but with two extra dependencies on the end. Thus I think adding the CacheItemPoolInterface after the MapperFactory rather than at the end of the constructor args might be handy to keep that symmetry.